### PR TITLE
Update tw.json

### PR DESCRIPTION
**Pull request by rhuba8324**
Changed Notes, Note, Security Reports to more accurate words.

I misunderstood the meaning for these lines, now fixed with more accurate and understandable words.

### CODE REVIEW

File: packages/locale/res/translations/tw.json [hunk 1]
```diff
@@ -2155,4 +2155,4 @@
     "Custom",
     "自訂"
   ]
-]
\ No newline at end of file
+]
```
Comment (BrunoBernardino) [inline packages/locale/res/translations/tw.json:2158]: @rhuba8324 can you try not adding this EOL, or running `npm run locale:extract` from the root of the repo to automatically fix it? It's fine if you can't, I'll just have to fix it after merging.

Comment (rhuba8324) [reply packages/locale/res/translations/tw.json]: It's now fixed, at least I think so.
Sorry for my mistake.

**Comment by rhuba8324** [comment]
EOL fixed, sorry for my mistake.
Still learning about these things.

File: packages/locale/res/translations/tw.json (hunk 1 above)
Comment (BrunoBernardino) [reply packages/locale/res/translations/tw.json]: No worries, not something you should know _a priori_!

**Pull request merged**